entries/u/ukpostbox.com.json

```diff
@@ -7,6 +7,9 @@
     ],
     "keywords": [
       "postal"
+    ],
+    "regions": [
+      "gb"
```

Comment:
I don't think this entry should have a `gb` region tag based on the website's description:

> Get a personal or business address in the UK regardless of where you are in the world. Read your post online or forward it internationally - a smarter post office.

Reply:
Ah my bad :see_no_evil: